cli_reference/cli_manager/cli-manager-using.adoc

```diff
@@ -0,0 +1,19 @@
+:_content-type: ASSEMBLY
+[id="cli-manager-install_{context}"]
+= Using the {cli-manager}
+include::_attributes/common-attributes.adoc[]
+:context: cli-manager-using
+
+toc::[]
+
+// Adding a plugin to the CLI Manager
+include::modules/cli-manager-adding-plugin-yaml.adoc[leveloffset=+1]
```

Comment:
This is the module I mentioned to pull out into its own assembly for admin tasks